dev/bin/camel-k-installer.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+camel_k_version="1.10.1"
```

Comment:
Hello @kostola @lucamolteni @robpblake  I was on PTO and missed this PR. But wouldn't it be better to add the kamel tool to the prerequisites in README instead of automatically downloading it? We expect user to install minikube, kubectl and other command line tools. So why the "kamel" is different? User may have his kamel tool already installed.